courses/free-courses-ar.md

```diff
@@ -186,19 +200,22 @@
 * [HTML And CSS Template 1&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAzHSjcR-HnW9tnxyuye8KbF) - Elzero Web School
 * [HTML And CSS Template 2&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAy1l-2A21ng3gxEyocruT0t) - Elzero Web School
 * [HTML And CSS Template 3&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAxuCSp2_-9LurPqRVwketnc) - Elzero Web School
+* [HTML Crash Course Tutorials for Beginners: HTML كامل للمبتدئين](https://www.youtube.com/watch?v=rytA8dLsSV8) - Mohammed Elzanaty
 * [Learn CSS In Arabic 2021&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAzjsz06gkzlSrlev53MGIKe) - Elzero Web School
 * [Learn HTML In Arabic 2021&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAw_t_XWUFbBX-c9MafPk9ji) - Elzero Web School
 * [Learn SASS In Arabic 2021&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAzlpyFHOaB3b-eubmF0TAV2) - Elzero Web School
+* [Learn Web Design From Scratch - HTML&rlm;](https://www.youtube.com/playlist?list=PLsECTUuTGe7oVwBXceqGWfpZQtdYz8hEh) - Techs Experts
 * [Sass&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAz6bF7qObm2a1mLN_WHAWQo) - Elzero Web School
 * [Sass - Create a website&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAz9sluuyOWPifXvySgrGma8) - Elzero Web School
-* [Yonisfy CS94's \| Web Programming with JavaScript&rlm;](https://www.youtube.com/playlist?list=PLLWuK602vNiXqkUqcoKrVjCyOrZh7Tw0e) - Mohammed Elzanaty
```

Comment:
There are any reason for removing this resource? Not is duplicated neither moved

Reply:
It was a playlist that contained one long video, so I replaced it with this 

In Line 203:
```
* [HTML Crash Course Tutorials for Beginners: HTML كامل للمبتدئين](https://www.youtube.com/watch?v=rytA8dLsSV8) - Mohammed Elzanaty
```

I suggest making a new postfix text called `Combined Video` for this situation

```
* [name](url) - name (Combined Video)
```

Because there are people making a 7 to 10 hours video to explain something (combined course)
Or delete the old course and merge it in one video

Reply:
> I suggest making a new postfix text called `Combined Video` for this situation

Really not is needed if we detect that chapters are integrated in timeline bar